projects/storefrontapp-e2e-cypress/cypress/integration/regression/asm/asm.e2e-spec.ts

```diff
@@ -81,6 +89,13 @@ context('Assisted Service Module', () => {
       cy.get('button[title="Close ASM"]').click();
       cy.get('cx-asm-main-ui').should('exist');
       cy.get('cx-asm-main-ui').should('not.be.visible');
+
+      // CXSPA-301/GH-14914
+      // Must ensure that site is still functional after service agent logout
+      navigateToHomepage();
+      cy.get('cx-storefront.stop-navigating').should('exist');
+      navigateToCategory('Brands', 'brands', false);
```

Comment:
See other comment about using force: true

Reply:
I made sure this is in a stable place before clicking so that force true is not needed.